src/randomNumber.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const randomNumber = () => {
+  const digits = [];
+  let result = '';
+
+  for (let i = 0; i < 10; i++) {
+    digits.push(i);
+  }
+
+  while (result.length < 4) {
+    const randomIndex = Math.floor(Math.random() * digits.length);
+
+    result += digits[randomIndex];
```

Comment:
Ensure that the generated number has 4 different digits. The current implementation may produce a number with repeated digits.